core/src/main/java/org/apache/iceberg/actions/SizeBasedDataRewriter.java

```diff
@@ -67,6 +68,8 @@ public Set<String> validOptions() {
   public void init(Map<String, String> options) {
     super.init(options);
     this.deleteFileThreshold = deleteFileThreshold(options);
+    this.deleteRatioThreshold = PropertyUtil.propertyAsDouble(
+            options, "delete-ratio-threshold", DEFAULT_DELETE_THRESHOLD);
```

Comment:
`delete-ratio-threshold` should be a static final (similar to `DELETE_FILE_THRESHOLD`) and have a proper javadoc documentation so that it's clear what this parameter is used for